### ISSUE TITLE
Unclear statement for the blank line for comment. (Bugzilla Bug 4160)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore-docs/edk2-CCodingStandardsSpecification#24]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 4160
Date: 2022-11-18T22:33:29+00:00
From: @changab
To:   @changab
CC:   @lgao4, @mdkinney

Last updated: 2022-11-29T21:54:02+00:00

**Comment by tianocore-issues** [comment]
## Comment 20064
Date: 2022-11-18 22:33:29 +0000
From: @changab

- Industry Specification: ---
- Release Observed: EDK II Master
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): @changab

1. Unclear statement for the blank line above/below a block of comment. Is this rule also applied to the one-line comment?

2. Some comment examples in CSS are not consistent with the spec. For example, the example in section 6.5.2.5.

**Comment by tianocore-issues** [comment]
## Comment 20136
Date: 2022-11-29 21:54:02 +0000
From: @lgao4

Abner is working on it.

**Comment by mdkinney** [comment]
@changab Has this issue been resolved?

**Comment by changab** [comment]
> [@changab](https://github.com/changab) Has this issue been resolved?

No, I didn't.  I think I forgot this. Let me take a look at it this week.

**Comment by changab** [comment]
PR here: https://github.com/tianocore-docs/edk2-CCodingStandardsSpecification/pull/28

**Comment by changab** [comment]
Fixed in https://github.com/tianocore-docs/edk2-CCodingStandardsSpecification/pull/29.
